en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+
+    !!! note
+        The **Choreo GitHub App** requires the following permissions:
+         - Read and write access to code and pull requests.
+         - Read access to issues and metadata.
+        
+        You can [revoke access](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-authorized-integrations#reviewing-your-authorized-github-apps) if needed. Write access is only used for sending pull requests; Choreo will not push changes directly to your repository.    
+
+5. Enter the following repository details:
+
+    | **Field**              | **Value**          |
+    |------------------------|--------------------|
+    | **Organization**       | Your GitHub account|
+    | **Repository**         | choreo-samples     |
+    | **Branch**             | **`main`**         |
+    | **API Directory**      | /choreo-samples/department-api-proxy-from-github |
+
+6. Specify the following values as API proxy details:
+
+    !!! info
+        The **Context** field must be unique and cannot be changed after creation.**Version** and **Target** are mandatory fields. **Target** can be changed at any time after the creation.
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Context**     | `department-service`                                   |
+    | **Version**     | `v1.0`                                      |
+    | **Target**      | `https://samples.choreoapps.dev/company/hr`|
+
+7. Specify the following values as component details:
+
+    !!! info
+        The **Component Name** field must be unique and cannot be changed after creation. This value is automatically generated, but you can edit it if necessary.
+        **Component Display Name** is a required field.
+
+    | **Field**       | **Value**                                  |
+    |-----------------|--------------------------------------------|
+    | **Component Display Name**| `Department Service`                    |
+    | **Component Name**        | `departmentService`                    |
+    | **Description**           | `This is a sample proxy for department service`     |    
+
+8. Click **Create**. This creates the API proxy component and takes you to the **Build** page.
+
+!!! note
+    When you create an API proxy from a GitHub repository source, the GitHub source serves as the single source of truth. Therefore, any modifications, such as adding or deleting resources, must be made through the GitHub repository.
+
+## Step 2: Build
+
+!!! info
+    An initial build starts automatically as soon as you create the API proxy.
+
+1. On the project home page, click on the `Department Service` component you created. This takes you to the component overview page.
```

Comment:
Since the user is taken to the build page on creating the component, we don't need these steps right? We can say perhaps just mention that "The initial build starts automatically as soon as you create the API proxy. The build process may take some time. You can track progress in the **Build Details** pane. Once complete, the build status changes to **Success**."

Reply:
@shanakama did we address this comment when resolving?

Reply:
I think we should keep this step. Even the user automatically redirect to the build page once after the creation completed , that is one time behavior. All other times , user should follow above steps to navigate to the build page.